rtdb-limit-child-nodes/functions/package.json

```diff
@@ -11,8 +11,8 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "firebase-admin": "^8.0.0",
-    "firebase-functions": "^3.13.2"
+    "firebase-admin": "^9.11.0",
```

Comment:
Updating these references causes the following build issues....

```js
src/index.ts(32,33): error TS2339: Property 'path' does not exist on type 'Reference'.
src/index.ts(43,39): error TS2339: Property 'path' does not exist on type 'Reference'.
src/index.ts(45,38): error TS2339: Property 'path' does not exist on type 'Reference'.
src/index.ts(47,36): error TS2339: Property 'path' does not exist on type 'Reference'.
```

It seems path was removed at some point before the `v9` release